server/models/DB.sql

```diff
@@ -14,15 +14,15 @@ CREATE TABLE users (
   password VARCHAR (100) NOT NULL,
   profile_pic VARCHAR DEFAULT 'userProfile.png',
   company VARCHAR (50) DEFAULT '',	
-  is_landlord BOOLEAN DEFAULT false NOT NULL
```

Comment:
Remove landlord_id from table - circular reference with landlords

Reply:
agreed. it may be a longer process, but better practice to reference only one way (i.e. users -> landlords or vice/versa)

Reply:
this may actually cause a problem with SQL as well, as you are not allowed to delete a table with a foreign key reference. this circular reference would prevent you from deleting either table, as they reference each other's primary keys

Reply:
removed!